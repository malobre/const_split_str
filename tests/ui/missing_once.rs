fn main() {
    let _ = const_split_str::split_once!("head-tail", "?");
}
