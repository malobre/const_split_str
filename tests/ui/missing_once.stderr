error: delimiter is not contained in the input string
 --> tests/ui/missing_once.rs:2:55
  |
2 |     let _ = const_split_str::split_once!("head-tail", "?");
  |                                                       ^^^
